protected/controllers/AuditController.php

```diff
@@ -28,18 +28,13 @@ public function accessRules()
     {
         return array(
             array('allow',
-                'roles' => array('OprnViewClinical'),
+                'roles' => array('admin'),
             ),
         );
     }
 
     public function beforeAction($action)
     {
-        Yii::app()->assetManager->registerScriptFile('js/audit.js');
-        $userid = Yii::app()->session['user']->id;
-        if (($userid != 2103) and ($userid != 122) and ($userid != 613) and ($userid != 1330) and ($userid != 1)) {
```

Comment:
Curiously, this same logic is also in DicomLogViewerController, but commented out. It even uses teh same user IDs